com.unity.multiplayer.mlapi/Runtime/Spawning/NetworkSpawnManager.cs

```diff
@@ -250,13 +275,12 @@ internal NetworkObject CreateLocalNetworkObject(bool softCreate, uint prefabHash
             }
             else
             {
-                // SoftSync them by mapping
+                // Check to see if the hash is in our pending in-scene objects waiting to be "soft synchronized"
                 if (!PendingSoftSyncObjects.ContainsKey(prefabHash))
                 {
-                    // TODO: Fix this message
                     if (NetworkLog.CurrentLogLevel <= LogLevel.Error)
                     {
-                        NetworkLog.LogError("Cannot find pending soft sync object. Is the projects the same?");
+                        NetworkLog.LogError($"{nameof(NetworkPrefab)} hash was not found! In-Scene placed NetworkObject soft synchronization failure for Hash: {prefabHash.ToString("X")}!");
```

Comment:
Similar to above, should we not exit when this happens?  Though maybe the caller receives the null and then exits.

Reply:
See the notes above for NetworkSceneManager.cs